### ISSUE TITLE
no way to set the path for ffmpeg

### ISSUE THREAD

**Issue by w2daemon** [norkunas/youtube-dl-php#238]
Hi! How can I set the path for ffmpeg? I'm getting this error on MacOs Sonoma: **ffprobe and ffmpeg not found. Please install or provide the path using --ffmpeg-location**.  The path for it is /usr/local/bin/ffmpeg. I'm able to set the path for yt-dlp. Same directory BTW.
Thanks!

**Comment by norkunas** [comment]
Hey.
You can set ffmpeg location via Options instance, as it's yt-dlp option.

**Comment by w2daemon** [comment]
Can you provide an example for that? Thank you!

**Comment by norkunas** [comment]
```
$yt->download(
    Options::create()
        ->ffmpegLocation(...)
        ->url(...)
); ```

**Comment by w2daemon** [comment]
Thank you! I will give it a try.

**Comment by w2daemon** [comment]
Yeah, it's working! Is there any way to compress the audio file? It is bigger than the webm file. What should I set for audio quality? I tried 1 and 2...but no change. 0 should be the best...Anyway, you can mark this issue as solved. Thank you!
